agent/api/task/task.go

```diff
@@ -69,9 +69,27 @@ const (
 	// container's network mode to be mapped to that of another existing container
 	networkModeContainerPrefix = "container:"
 
+	// awslogsType is the type of AWS logs specified in the log configuration
+	awslogsType = "awslogs"
+
 	// awslogsCredsEndpointOpt is the awslogs option that is used to pass in an
 	// http endpoint for authentication
 	awslogsCredsEndpointOpt = "awslogs-credentials-endpoint"
+	// awslogsRegionOpt is the awslogs option to specify the AWS logs region
+	awslogsRegionOpt = "awslogs-region"
+	// awslogsEndpointOpt is the awslogs option to specify the AWS logs endpoint
+	awslogsEndpointOpt = "awslogs-endpoint"
+	// awslogsServiceName is the service name of AWS logs, it's used to
+	// compose the AWS logs endpoint
+	awslogsServiceName = "logs"
+
+	// The minimum docker CE version that supports "awslogs-endpoint" log option.
+	// Will update this when we know the version.
```

Comment:
How will you remember to update this?

Reply:
I won't merge this PR until I know the versions, and you don't need to approve it now, I will ask you (and someone else) to review this again when I am ready to merge.